spec/locale.html

```diff
@@ -350,4 +350,35 @@ <h1>get Intl.Locale.prototype.region</h1>
       </emu-alg>
     </emu-clause>
   </emu-clause>
+
+  <emu-clause id="sec-properties-of-intl-locale-instances">
+    <h1>Properties of Intl.Locale Instances</h1>
+
+    <p>
+      Intl.Locale instances are ordinary objects that inherit properties from %Locale.prototype%.
+    </p>
+
+    <p>
+      Intl.Locale instances have an [[InitializedLocale]] internal slot.
+    </p>
+
+    <p>
+      Intl.Locale instances also have several internal slots that are computed by the constructor:
+    </p>
+
+    <ul>
+      <li>[[Locale]] is a String value with the language tag of the locale whose localization is used for formatting.</li>
+      <li>[[Calendar]] is a String value with the *"type"* given in <a href="https://unicode.org/reports/tr35/#Key_And_Type_Definitions_">Unicode Technical Standard #35 Part 1 Core, Section 3.6.1 Key and Type Definitions</a> for the calendar used for formatting.</li>
```

Comment:
" or *undefined*".
Notice, this is different from DateTimeFormat because in DateTimeFormat  `[[Calendar]]`  is from the result of ResolveLocale() which could be all String, but in here the  `[[Calendar]]` could be *undefined* not a String, right.
For example, if we have the following

```
let l = new Intl.Locale("en");
l.calendar; // This should be undefined
```